src/app/WebViewImpl.qml

```diff
@@ -248,35 +248,43 @@ WebView {
          }
     }
 
-      onCertificateError: function(certificateError) {
-
-          certificateError.defer()
-          var certificateVerificationDialog = PopupUtils.open(Qt.resolvedUrl("CertificateVerificationDialog.qml"), this);
-          certificateVerificationDialog.host = UrlUtils.extractHost(certificateError.url);
-          certificateErrorsMap[certificateVerificationDialog.host] = certificateError
-          certificateErrorsMapChanged()
-          certificateVerificationDialog.localizedErrorMessage = certificateError.description
-          certificateVerificationDialog.errorIsOverridable = certificateError.overridable
-          certificateVerificationDialog.accept.connect(certificateError.ignoreCertificateError)
-          certificateVerificationDialog.reject.connect(certificateError.rejectCertificate)
-      }
+    // returnIfPossible -> rejects bad certificate and returns to last page
+    function returnIfPossible(certificateError){
+        // deferred rejection so that c.v.Dialog isn't called twice
+        certificateError.rejectCertificate
```

Comment:
this line shows `Type Error: Cannot read property 'rejectCertificate' of undefined` in the logfile.
Since QtCreator was showing a warning, I changed it to `certificateError.rejectCertificate()`
But then it was still not working: `TypeError: Cannot call method 'rejectCertificate' of undefined`
so in fact "certificateError" is not passed to the function "returnIfPossible", and does nothing

Reply:
... but before putting more effort into it, we should discuss with Dalton if maybe a Cancel button is enough, and we could instead add the back button to the error sheet instead. (https://github.com/ubports/morph-browser/issues/116)

Reply:
So for some reason `certificateError` isn't getting passed to the function.

Reply:
I think only arguments passed to the reject() signal of the dialog would get passed to the function